### ISSUE TITLE
LC0033 runtime version clarification

### ISSUE THREAD

**Issue by fvet** [StefanMaron/BusinessCentral.LinterCop#361]
While prepping our apps for BC23 (using application 23.1.0.0, since it has some publishers we require) and setting the runtime to 12.0, I get a LC0033 warning that runtime should be set to 12.**1**.

However, neither the intelisense, neither the documentation list 12.**1** as a support version.

https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/devenv-choosing-runtime

Am I missing something that application 23.1.0.0 require runtime 12.**1**?

![image](https://github.com/StefanMaron/BusinessCentral.LinterCop/assets/25268332/3c2b4c21-9352-40ba-832f-328180a2183c)

_PS: In above screenshot, my runtime was set to "12.0" (changed to 12. to get the intellisense)_

**Comment by Arthurvdv** [comment]
![image](https://github.com/StefanMaron/BusinessCentral.LinterCop/assets/44637996/68f4dff5-c511-4a2f-8522-57d0de466e9e)
In the AL Language v12.3.900902 this should be the latest version.

I also noticed that the intelliSense doesn't suggest this version, but I am able to set this to 12.1 and compile without any warnings/errors on the runtime version setting.

It could be that the intelliSense only shows all the version up to the CurrentRelease. I could improve the rule on that. The downside is that today the rule will suggest 12.0 and when 21.1 becomes the CurrentRelease, the rule wil then suggest the change it to 12.1, even when nothing else has been changed on the project. This makes the rule a bit less predictable?
